lib/stoplight/default.rb

```diff
@@ -23,5 +23,7 @@ module Default
     ].freeze
 
     THRESHOLD = 3
+
+    TIME_FORMAT = '%Y-%m-%dT%H:%M:%S.%N%:z'.freeze
```

Comment:
The `Default` module is meant for things that can be changed by the user. Would you mind moving this constant into the `Failure` module?